src/main/java/com/gluonhq/richtextarea/RichTextAreaSkin.java

```diff
@@ -262,8 +264,8 @@ protected RichTextAreaSkin(final RichTextArea control) {
 
         caretChangeListener = (obs, ov, nv) -> viewModel.getParagraphWithCaret()
                 .ifPresent(paragraph -> Platform.runLater(paragraphListView::scrollIfNeeded));
+        focusListener = o -> paragraphListView.updateLayout();
```

Comment:
Is rebuilding all cells really required for focus listener ?

Reply:
Since each cell now has control of its own caret, we need to trigger `cell::updateItem` that has:
`paragraphTile.updateLayout();` which ultimately calls `updateCaretPosition` to start/stop the caret animation.